sympy/printing/fcode.py

```diff
@@ -129,6 +131,33 @@ def _lead(self):
         else:
             raise ValueError("Unknown source format: %s" % self._settings['source_format'])
 
+    def _print_Symbol(self, expr):
+        if self._settings['name_mangling'] is True:
```

Comment:
`is True` is problematic (e.g. `(numpy.array([1]) == 1)[0] is True` returns `False` for example)